docker-compose.yml

```diff
@@ -19,6 +19,7 @@ services:
       EMAIL_PASS: ${EMAIL_PASS}
       EMAIL_RECEIVER: ${EMAIL_RECEIVER}
       FE_URL: ${FE_URL}
+      BE_URL: ${BE_URL}
```

Comment:
why we need be url in docker-compose ?

Reply:
cos parameters are provided to the application when running `docker-compose` so we need to pass then into the scope